java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2590,15 +2590,31 @@ public void objectFreq_returns_null(BaseClient client) {
     @MethodSource("getClients")
     public void objectFreq(BaseClient client) {
         String key = UUID.randomUUID().toString();
+        String maxmemoryPolicy = "maxmemory-policy";
+        String allkeysLfu = "allkeys-lfu";
+
         if (client instanceof RedisClient) {
-            assertEquals(
-                    OK, ((RedisClient) client).configSet(Map.of("maxmemory-policy", "allkeys-lfu")).get());
+            RedisClient redisClient = (RedisClient) client;
```

Comment:
maybe split into 2 test cases and move them to corresponding files? Standalone client and cluster client.